custom_components/ramses_cc/remote.py

```diff
@@ -142,6 +154,11 @@ async def async_learn_command(
         if command[0] in self._commands:
             await self.async_delete_command(command)
 
+        # If packet_string is provided, add it directly without listening
+        if packet_string:
```

Comment:
Not a problem, but a risk, think about adding a warning or extra confirmation step, making users aware they can upset their hardware